bouncer/src/repo/manipulator/modelconvertor/import/repo_model_import_assimp.cpp

```diff
@@ -1330,6 +1330,8 @@ bool AssimpModelImport::SetRootOrientationFromMetadata()
 	reqMetadataExists &= assimpScene->mMetaData->Get<int32_t>("FrontAxisSign", frontAxisSign);
 	reqMetadataExists &= assimpScene->mMetaData->Get<int32_t>("CoordAxis", coordAxis);
 	reqMetadataExists &= assimpScene->mMetaData->Get<int32_t>("CoordAxisSign", coordAxisSign);
+	repoInfo << "reqMetadataExists=" << (reqMetadataExists) ? "true" : "false";
```

Comment:
please remove debugging

Reply:
As per your suggestion removed the debugging logs.